docs/cmake-presets.md

```diff
@@ -25,29 +25,30 @@ A new setting, `cmake.useCMakePresets`, has been added to `settings.json`:
+**CMake: Select Configure Preset** lists the union of non-hidden Configure Presets defined in `CMakePresets.json` and `CMakeUserPresets.json`. Select a Configure Preset to make it the active Configure Preset. This is the `configurePreset` value that's used when CMake is invoked to generate the project build system. The active Configure Preset appears on the status bar.
 
-CMake Tools uses the value of `hostOS` in the Microsoft Visual Studio Settings vendor map to hide Configure Presets that don't apply to your platform. See the entry for `hostOS` in the table under [Visual Studio Settings vendor map](cmake-presets-json-reference.md#visual-studio-settings-vendor-map) for more information.
+CMake Tools uses the value of `hostOS` in the Microsoft Visual Studio Settings vendor map to hide Configure Presets that don't apply to your platform. For more information, see the entry for `hostOS` in the table under [Visual Studio Settings vendor map](cmake-presets-json-reference.md#visual-studio-settings-vendor-map).
 
 ### CMake: Configure
 
 To configure the project, run **CMake: Configure** from the command palette. This is the same as running `cmake --preset <configurePreset>` from the command line, where `<configurePreset>` is the name of the active Configure Preset.
 
-> Note: Windows developers, you must launch VS Code from a developer command prompt, or run the **CMake: Scan for Compilers** command before the CMake Tools extension can detect build tools (CMake, Ninja) that are installed with Visual Studio.
+> [!NOTE]
+> If you're a windows developer, you must open Visual Studio Code from a developer command prompt. Or, run the **CMake: Scan for Compilers** command before the CMake Tools extension can detect build tools (CMake, Ninja) that are installed with Visual Studio.
 
 ### CMake: Select Build Preset
 
-**CMake: Select Build Preset** lists the Default Build Preset and the union of non-hidden Build Presets defined in `CMakePresets.json` and `CMakeUserPresets.json`. The Default Build Preset is equivalent to passing `cmake --build` with no additional arguments from the command line. Select a Build Preset to make it the active Build Preset. This is the `buildPreset` used when CMake is invoked to build the project. The active Build Preset is displayed in the status bar.
+**CMake: Select Build Preset** lists the default Build Preset and the union of non-hidden Build Presets defined in `CMakePresets.json` and `CMakeUserPresets.json`. The default Build Preset is equivalent to passing `cmake --build` with no additional arguments from the command line. Select a Build Preset to make it the active Build Preset. This is the `buildPreset` value that's used when CMake is invoked to build the project. The active Build Preset is displayed on the status bar.
```

Comment:
I'm not sure on the casing for default Build Preset. @esweet431, is Default Build Preset how the CMake docs refer to this? In other words, is 'Default Build Preset' a thing, or are we talking about the default `Build Preset`?

Reply:
The "Default" Build Preset is a VS-specific concept. It's name is "Default" (that's how it shows up to users) so I used the same casing for the docs.

Reply:
Understood. I just reverted the change.